.github/workflows/build-docs.yml

```diff
@@ -104,11 +106,11 @@ jobs:
         env:
           GITHUB_HEAD_SHA: ${{ github.event.pull_request.head.sha }}
 
+      # Commits to pull requests result in preview deployments
       - name: Publish Preview to Vercel
         uses: amondnet/vercel-action@v25
         if: |
-          github.event_name == 'pull_request' ||
```

Comment:
@neverett note that removing the `docs-prod` target might make it difficult for us to update the currently live version of docs.dagster.io.

Reply:
Hm that's not ideal, why is that? Would we need to update the current release branch instead?